gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -222,6 +222,9 @@ const CssVariablesPage = () => {
           <PropertyDocItem propertyVar="--nx-box-shadow-dropdown">
             The standard box-shadow to apply to dropdown elements.
           </PropertyDocItem>
+          <PropertyDocItem propertyVar="--nx-box-shadow-drawer">
+            The standard box-shadow to apply to dropdown elements.
```

Comment:
The variable name says "drawer" but the description says "dropdown"

Reply:
>The variable name says "drawer" but the description says "dropdown"

Based on the changes made in [7e25ef6](https://github.com/sonatype/sonatype-react-shared-components/pull/1097/commits/7e25ef66b14f6de8c1d93ad316da5864a951b0cb), variable documentation updated here: [f72177f](https://github.com/sonatype/sonatype-react-shared-components/pull/1097/commits/f72177f4d926bc0ccd6d23a6d1ffb68cc0da92e6)